content/blog/2025/04/2025-04-28-election-announcement.md

```diff
@@ -0,0 +1,97 @@
+
+![A visual election timeline for the next few years: in 2024 we elect all constituencies, in 2025 we elect Gold, Silver, Individual, and Associate Members, in 2026 we elect Platinum, Ecosystem, Guardians, and Spec Core Team Members, in 2027 we elect Gold, Silver, Individual, and Associate Members, and so on.](/blog/img/2024-election-years.png)
+
+There are currently no term limits, but term limits are a best practice in open governance so we intend to introduce them as we find our collective footing.
+
+## Who can nominate
+
+You must be a member of one of the constituency groups in order to nominate yourself or someone else to be a candidate in the election. By far our largest constituency group are Individual Members, who number in the 100’s. If you have donated at least $60 USD to the Foundation since April 20, 2024 on Donorbox, then you are eligible to nominate and vote.
+
+All nominees must have consented to nomination _before being nominated_ and must be a community member in good standing. That means they must have a clean track record with respect to our [Code of Conduct](https://matrix.org/legal/code-of-conduct/). We want our Governing Board to be a clearing house for a variety of priorities and perspectives, and to be able to hold space for each other even when there are disagreements.
+
+## Who can vote
+
+All members of each constituency group are entitled to vote on the candidates within that constituency group. We will be reaching out to everyone we have contact information for, to ensure we have our voter rolls together before voting begins at the end of May.
+
+We will hold an election in each of the four constituencies that are up for election this year, even if a given constituency has fewer candidates than there are seats allocated for them on the Governing Board. This is, effectively, a vote of confidence.
+
+If you believe you are eligible to participate but have not heard from us or [OpaVote](https://www.opavote.com/) – the election system we have chosen for this year’s elections – by May 24th, [please email us promptly](mailto:elections@foundation.matrix.org). We build our voter rolls on May 31st and can't change them after that!
+
+All of the candidates will be listed on an elections page here on our website which we’ll publish on May 17th.
+
+## How to prepare yourself
+
+If you want to be a candidate in the election, there are a few things you should know:
+
+* All candidates will be listed on our website, grouped by the constituency group.
+* We do not require people to use their “legal name.”
+* Candidates who are nominated by an organizational member will have their affiliation listed.
+* We expect candidates to provide a short bio and a short statement, collectively covering qualifications and priorities. Candidates may also provide a link to an extended statement or platform on their personal blog or website.
```

Comment:
we could link to last year's example https://matrix.org/foundation/governing-board-elections/